### ISSUE TITLE
FocalTverskyLoss produces nan loss value in loss_functions.py

### ISSUE THREAD

**Issue by j-c-carr** [j-c-carr/lightcones#13]
When computing the loss function, `FocalTverskyLoss` is returning nan values. I tried adding a small value to the loss as recommended [here](https://www.reddit.com/r/MLQuestions/comments/az1cc0/tensorflow_nan_loss/), but with no luck. [Here](https://stackoverflow.com/questions/33712178/tensorflow-nan-bug) is another thread about it.

**Comment by j-c-carr** [comment]
This was a problem with indentation of the code inside the `FocalTverskyLoss` function definition.
